usr/lib/modprobe.d/50-playtron.conf

```diff
@@ -0,0 +1,2 @@
+blacklist nouveau
+options nvidia-drm modeset=1
```

Comment:
Do we also want to set the `fbdev=1` option for the NVIDIA driver here?

NVIDIA said they will finally be setting both `modeset=1` and `fbdev=1` by default in an upcoming driver release but we'll still need these workarounds short-term.

https://www.phoronix.com/news/NVIDIA-Wayland-Roadmap-2024

Reply:
I thought about it, but since we didn't really test with it, I was erring on the safe side and not including it.
On the other hand, if we wanted to include it now would be the time.